src/bundle/Resources/views/notifications/notifications_modal_body.html.twig

```diff
@@ -15,7 +15,7 @@
     <tbody class="n-table__body">
     {% if pager.count is same as(0) %}
         <tr>
-            <td colspan="4">
+            <td colspan="3">
```

Comment:
Note: it broke down after changing `table-layout` to `fixed` (https://github.com/ezsystems/ezplatform-admin-ui/pull/951/files#diff-c7c7f04732ef05086f0a5fcd08f19a3bR31), because table has 3 columns.